src/diagram.ts

```diff
@@ -238,6 +240,31 @@ export class Diagram {
         }
     }
 
+    undisplayNodes(): void {
+        // reset undisplayed nodes when escape key pressed
+        if ((event as KeyboardEvent).keyCode === 27) {
+            const node = d3.selectAll('.node')
+            node.each(function (d) {
+                d.displayCallback(this);
+            });
+
+            const link = d3.selectAll('.link')
+            link.each(function (d) {
+                d.displayCallback(this);
+            });
+        } else {
```

Comment:
Can we specify a single key ( eg: "d" ) to delete nodes for the same reason as  src/diagram.ts#173?

Reply:
Very nice idea. Done.